### ISSUE TITLE
ModuleNotFoundError issue with PyPDF2==2.10.1

### ISSUE THREAD

**Issue by JaMe76** [py-pdf/PyPDF2#1242]
When importing

```
from PyPDF2 import PdfFileReader, PdfFileWriter
```

I get an immediate error

```
Traceback (most recent call last):
  File "~/dummy.py", line 21, in <module>
    from PyPDF2 import PdfFileReader, PdfFileWriter
  File "~/venv/lib/python3.8/site-packages/PyPDF2/__init__.py", line 10, in <module>
    from ._encryption import PasswordType
  File "~/venv/lib/python3.8/site-packages/PyPDF2/_encryption.py", line 36, in <module>
    from PyPDF2.generic import (
ModuleNotFoundError: No module named 'PyPDF2.generic'
```
This occurs with the latest 

pip install PyPDF2==2.10.1

The older version works as expected:

pip install PyPDF2==2.10.0

## Environment

Which environment were you using when you encountered the problem?

```bash
$ python -m platform
Linux-5.15.0-46-generic-x86_64-with-glibc2.29

$ python -c "import PyPDF2;print(PyPDF2.__version__)"
File "<string>", line 1, in <module>
  File "~/venv/lib/python3.8/site-packages/PyPDF2/__init__.py", line 10, in <module>
    from ._encryption import PasswordType
  File "~/venv/lib/python3.8/site-packages/PyPDF2/_encryption.py", line 36, in <module>
    from PyPDF2.generic import (
ModuleNotFoundError: No module named 'PyPDF2.generic'
```

## Code + PDF

```
from PyPDF2 import PdfFileReader, PdfFileWriter
```

Share here the PDF file(s) that cause the issue. The smaller they are, the
better. Let us know if we may add them to our tests! -> No PDF required

## Traceback

This is the complete Traceback I see:

```
 File "<string>", line 1, in <module>
 File "~/venv/lib/python3.8/site-packages/PyPDF2/__init__.py", line 10, in <module>
 from ._encryption import PasswordType
 File "~/venv/lib/python3.8/site-packages/PyPDF2/_encryption.py", line 36, in <module>
 from PyPDF2.generic import (
ModuleNotFoundError: No module named 'PyPDF2.generic'
```

**Comment by KJD-II** [comment]
I'm seeing the same error with 3.8. The package is missing the `generic.py` file in 2.10.1, present in 2.10.0

**Comment by MartinThoma** [comment]
Duplicate of #1241 

I'm sorry for the hickup; I forgot to add the submodule to the distribution. It's fixed with `PyPDF2==2.10.2`

**Comment by MartinThoma** [comment]
@JaMe76 Thank you for reporting the issue so quickly :heart: 

If you want to, I would add you as a contributor: https://pypdf2.readthedocs.io/en/latest/meta/CONTRIBUTORS.html
